Model/Config/Backend/Tokenization.php

```diff
@@ -150,6 +152,15 @@ private function buildPaymentMethodArray(): array
                     'recurring_processing_model' => null
                 ];
 
+                $arrayKey = $this->mathRandom->getUniqueHash('_');
+                $result[$arrayKey] = $paymentMethodValue;
+            } elseif ($adyenPaymentMethod === AdyenPosCloudConfigProvider::CODE ){
```

Comment:
With the latest changes in `rc-v9`, this block will create a confliction. Could you please check the latest implementation of this block [here](https://github.com/Adyen/adyen-magento2/blob/31ff703486c930140efdf5503ff86868622e7556/Model/Config/Backend/Tokenization.php#L128) and modify the code? After implementing the new block, you need to specify `<supports_recurring>1</supports_recurring>` in payment methods config block in [config.xml](https://github.com/Adyen/adyen-magento2/blob/31ff703486c930140efdf5503ff86868622e7556/etc/config.xml#L378).